pkg/kubelet/dockershim/remote/docker_service.go

```diff
@@ -49,6 +49,10 @@ func (d *dockerService) Version(ctx context.Context, r *runtimeApi.VersionReques
 	return d.runtimeService.Version(r.GetVersion())
 }
 
+func (d *dockerService) Status(ctx context.Context, r *runtimeApi.StatusRequest) (*runtimeApi.StatusResponse, error) {
+	return nil, fmt.Errorf("not implemented")
```

Comment:
Please add godoc. how are we planning to integrate this into the `defaultNodeStatusFuncs`? I'm guessing it needs to make its way into `runtimeState.networkErrors`?